doc/index.rst

```diff
@@ -35,6 +35,7 @@ or using ``conda``
    examples/introduction
    EarthCube 2021 demo <https://earthcube2021.github.io/ec21_book/notebooks/ec21_cherian_etal/DC_01_cf-xarray.html>
    Ocean model demo <https://pop-tools.readthedocs.io/en/latest/examples/cesm-cmip-cf-xarray.html>
+   Writing model agnostic code (COSIMA) <https://cosima-recipes.readthedocs.io/en/latest/tutorials/Model_Agnostic_Analysis.html#gallery-tutorials-model-agnostic-analysis-ipynb>
```

Comment:
```suggestion
   CESM Ocean model demo <https://pop-tools.readthedocs.io/en/latest/examples/cesm-cmip-cf-xarray.html>
   COSIMA ocean/ice model demo <https://cosima-recipes.readthedocs.io/en/latest/tutorials/Model_Agnostic_Analysis.html#gallery-tutorials-model-agnostic-analysis-ipynb>
```

Reply:
Thanks @aidanheerdegen does the suggestion seem accurate to you?

Reply:
perhaps "COSIMA ocean-sea ice model demo"?

Reply:
Agreed that is better. Have made that change and also made the CESM case consistent.